test/integration/services/alert.js

```diff
@@ -62,57 +62,63 @@ context('Alert Service', function() {
     cy
       .get('.alert-box')
       .should('not.exist');
+  });
+
+  specify('Closing via dismiss button', function() {
+    const onComplete = cy.stub();
 
     cy
-      .clock()
-      .then(clock => {
-        clock.restore();
-      });
+      .clock();
 
-    const options = {
-      onUndo: cy.stub(),
-      onComplete: cy.stub(),
-    };
+    cy
+      .routesForDefault()
+      .visit();
 
     cy
       .getRadio(Radio => {
-        Radio.request('alert', 'show:undo', options);
+        Radio.request('alert', 'show:undo', { onComplete });
       });
 
-    cy
-      .clock();
-
     cy
       .get('.alert-box')
       .find('.js-dismiss')
       .click()
-      .click()
       .then(() => {
-        expect(options.onComplete).to.be.calledOnce;
+        expect(onComplete).to.be.calledOnce;
       })
       .tick(1000);
 
     cy
       .get('.alert-box')
       .should('not.exist');
+  });
+
+  specify('Closing via undo button', function() {
+    const onUndo = cy.stub();
 
     cy
-      .clock()
-      .then(clock => {
-        clock.restore();
-      });
+      .clock();
+
+    cy
+      .routesForDefault()
+      .visit();
 
     cy
       .getRadio(Radio => {
-        Radio.request('alert', 'show:undo', options);
+        Radio.request('alert', 'show:undo', { onUndo });
       });
 
     cy
       .get('.alert-box')
       .find('.js-undo')
       .click()
       .then(() => {
-        expect(options.onUndo).to.be.calledOnce;
-      });
+        expect(onUndo).to.be.calledOnce;
+      })
+      .tick(1000);
+
+    cy
+      .get('.alert-box')
+      .should('not.exist');
```

Comment:
I added this. Just checks that the alert is no longer there after the undo button is clicked. Before we were only checking that the radio was sent/called.